src/apps/chifra/internal/abis/options.go

```diff
@@ -136,6 +137,16 @@ func GetOptions() *AbisOptions {
 }
 
 func ResetOptions() {
+	// We want to keep writer between command file calls if
+	// the global format is JSON
+	w := GetOptions().Globals.Writer
 	defaultAbisOptions = AbisOptions{}
 	globals.SetDefaults(&defaultAbisOptions.Globals)
+	if GetOptions().Globals.Format == "json" || GetOptions().Globals.ShowRaw {
```

Comment:
Correct me if I'm wrong, but I think this test appears in numerous other places, and its purpose is basically to say "the following section of code is 'different' in the sense that it needs to 'close'" with "close" meaning that it must write the closing bracket for JSON, but in the case of a custom format could mean "put a date on the bottom of the custom built string that is the bank statement."

Can we make this test (wherever it appears) be a routine called something like `needsFooter` (that why I used `RenderHeader` and `RenderFooter` in the other code.